build/dsc.rb

```diff
@@ -4,11 +4,15 @@
 require 'pathname'
 require 'erb'
 require 'open-uri'
-require 'mof'
-require 'cim'
 require 'find'
 require 'logger'
-require 'charlock_holmes/string'
+
+# Bundler group 'build'
+if Bundler.rubygems.find_name('mof').any?
```

Comment:
On second thought, that would need puppet required. Is there some other way to determine we are on Windows and shield this?